sql/driving_distance/withPointsDD.sql

```diff
@@ -1,8 +1,13 @@
 /*PGR-GNU*****************************************************************
 File: withPointsDD.sql
 
+Generated with Template by:
+Copyright (c) 2015 pgRouting developers
+Mail: project@pgrouting.org
```

Comment:
Please also change this to `project at pgrouting.org` (also for other files above)